mffpy/header_block/helpers.py

```diff
@@ -0,0 +1,25 @@
+import struct
+from os import SEEK_CUR
+from typing import IO, Union
+from io import FileIO
+
+FileLike = Union[IO[bytes], FileIO]
+
+
+def read(fp: FileLike, format_str: str):
+    """read data from `fp` specified by `format_str`"""
+    num_bytes = struct.calcsize(format_str)
+    byts = fp.read(num_bytes)
+    ans = struct.unpack(format_str, byts)
```

Comment:
This line is causing a `mypy` error:
```
mypy --ignore-missing-imports mffpy
mffpy/header_block/helpers.py:13: error: Argument 2 to "unpack" has incompatible type "Optional[bytes]"; expected "Union[array[int], bytes, bytearray, memoryview]"
make: *** [test] Error 1
```
I'm not sure why that didn't get caught in the github workflow...

Reply:
I think you are using the wrong/right version of mypy.

Reply:
Yep, good thought. I don't get the error with `mypy==0.800`. Looks like that's what gets installed if you don't specify an exact version number.